web/pages/feeds/add.svelte

```diff
@@ -52,6 +53,7 @@
               current={$previewCandidate === candidate.url}
               on:click={() => {
                 setPreviewCandidate(candidate.url)
+                showSecondStep()
```

Comment:
We should put maximum code to the `core`.

Let’s put it to `setPreviewCandidate`.